src/binding/custom_mapping.txt

```diff
@@ -0,0 +1,6 @@
+LIS_KIND_MAP:
```

Comment:
I can add notes to `doc/notes/binding_c.txt`. I also can add a short comment in `src/binding/custom_mapping.txt`. But let me also explain it here:

MPI forum repo uses "kind" to categorize each parameter, names such as `COMMUNICATOR` and `POLYXFER_NUM_ELEM_NNI`. Then it uses mapping, one for each language binding, to map the "kind" to "type". For example, `COMMUNICATOR` is mapped to `MPI_Comm`, and `POLYXFER_NUM_ELEM_NNI` is mapped to `int` or `MPI_Count` depending on embiggenning flavor.

There are a few `MPIX_` functions with mpich-specific parameters, such as "general request class" and "gpu device type". Because they are not in the standard, so we have to supply our own "KIND" and "mapping". 

The kind is used following each parameter in `mpi_standard_api.txt` or in our config files such as `src/binding/c/request_api.txt` for `MPIX_` functions. The mapping is in `src/binding/apis_mapping.txt` for standard upstream mapping, and in `custom_mapping.txt` for custom mappings (for mpix parameters)

Reply:
Comments and notes added to both `doc/notes/binding_c.txt` and `src/binding/custom_mapping.txt`